backend/manager/modules/bll/src/main/java/org/ovirt/engine/core/bll/MaintenanceVdsCommand.java

```diff
@@ -157,6 +157,7 @@ protected ActionReturnValue migrateAllVms(ExecutionContext parentContext, boolea
     private ActionReturnValue migrateVms(List<VM> vms, ExecutionContext parentContext) {
         if (vms.isEmpty()) {
             ActionReturnValue returnValue = new ActionReturnValue();
+            returnValue.setValid(true);
```

Comment:
I think it's ok to assume that if succeeded==true then valid==true as well
we don't create new instance ActionReturnValue very often but when [we do](https://github.com/oVirt/ovirt-engine/blob/master/frontend/webadmin/modules/frontend/src/main/java/org/ovirt/engine/ui/frontend/communication/GWTRPCCommunicationProvider.java#L478-L479), we assume it's enough to set succeeded to true in order to reflect that the command's execution completed successfully

Reply:
I see. OK.